docs/unit.md

```diff
@@ -4,24 +4,71 @@ When running unit tests you should isolate your code so much that it could run i
 ## CLI commands
 A set of CLI commands are included in after-work.js to execute different test runners and tools.
 
-### aw-test-coverage (node.js)
-`aw-test-coverage` executes **Istanbul** to instrument the code and run tests to create a report of the current test coverage.
+### aw-test-runner (node.js)
+`aw-test-runner` is the default test runner for running unit tests with **Mocha** (using node.js). Appending the argument **cover** will trigger nyc to
+instrument the code before running the tests and make it possible to get test coverage.
 ```
-aw-test-coverage -- test/unit
+aw-test-runner [cover] ./test/unit
 ```
+
 ##### Arguments
-  * `-- "folder"`: The starting point to run code coverage upon.
-  * `-x "folder"`: Excludes this folder from the code coverage.
-  * `--dir "folder"`: Specifies the output folder for the report.
+  * `cover`: This will instrument the code and report the test coverage
+  * `-w`: Watch files for changes which triggers a re-run. (coverage will only be reported after finished test run)
+  * `--opts <file>`: path for mocha options file
+  * `--debug`: The script will 'break' on the very first line. Use the url chrome://inspect to attach to the debug session. It will also show the exact command that will be executed with all arguments
```

Comment:
The script 'breaks' on the very first line. The debug session is attached with URL: chrome://inspect. Also shows the exact command, including all arguments, that is executed.